cfpb/sbl-frontend
# feat: [Page] Authenticated Landing

## Changes

- Creates authenticated landing page
- Renders AuthenticatedLanding component at `/landing` route
- Routes users to authenticated landing page upon login

## How to test this PR

1. `yarn start`
2. Click "LOGIN" or "Sign in..."
3. Observe that you end up on the Authenticated Landing page
4. Click "File your small business lending data" and verify you are routed to the Filing app

## Screenshots
![screencapture-localhost-8899-landing-2023-12-21-16_00_49](https://github.com/cfpb/sbl-frontend/assets/2592907/443b7329-ded4-4373-a9aa-aa6619cda1cc)


## Notes
- TODO: Connect institution's `NAME | LEI` links to the View Institution Profile page once it's available
- TODO: Expand the type signature of the ListItem component in DSR to support more variants than just "string"
